tests/components/simplefin/test_sensor.py

```diff
@@ -0,0 +1,39 @@
+"""Sensor tests."""
+
+
+from unittest.mock import AsyncMock, patch
+
+import pytest
+from simplefin4py.model import AccountType
+
+from homeassistant.components.simplefin.coordinator import (
+    SimpleFinDataUpdateCoordinator,
+)
+from homeassistant.components.simplefin.sensor import SimpleFinBalanceSensor
+
```

Comment:
I'd suggest adding a snapshot test to test the state of your sensors

Reply:
I can't figure that out -> will do in a later PR